ext/vulnsrc/manual/manual.go

```diff
@@ -13,168 +12,7 @@ type updater struct {
 }
 
 // Vulnerabilities lists vulnerabilities which may not already exist in the feeds for other distros.
-var Vulnerabilities = []database.Vulnerability{
```

Comment:
These entries are no longer required, as we can fetch their entries from NVD properly now